guides/getting-started.md

```diff
@@ -12,10 +12,21 @@ account
 * A NAT router so EC2 instances in the private subnet can reach the internet
 * A security group for the NAT router so you can control access to it
 
-To get started, open a new file named "Cloudfile" and put the following Ruby
-code in it.
+To get started, create the following directory structure for your project.
+```
+my-convection-project/
+├── clouds
+│   └── us-east-1
+└── templates
+```
+In the top level of your convection project create a file "Gemfile" with the following inside.
+```ruby
+gem 'convection'
+```
 
+In the us-east-1 folder open a new file named "Cloudfile" and put the following Ruby
 ```ruby
+require_relative './templates/vpc.rb'
```

Comment:
@bcall-r7 does this work with the proposed layout? Does it still work if you move directories by a level? I think you should use `require_relative './templates.rb'` and load further templates from there with `Dir.glob { |file| require_relative './templates/#{file}' }`